lib/config/presets/internal/replacements.ts

```diff
@@ -901,6 +902,18 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'standard-version-to-commit-and-tag': {
+    description:
+      '`standard-version` is now maintained as `commit-and-tag-version`.',
```

Comment:
The [`conventional-changelog/standard-version` readme](https://github.com/conventional-changelog/standard-version/blob/master/README.md) says:

> **`standard-version` is deprecated**. If you're a GitHub user, I recommend [release-please](https://github.com/googleapis/release-please) as an alternative. If you're unable to use GitHub Actions, or if you need to stick with `standard-version` for some other reason, you can use the [commit-and-tag-version]( https://github.com/absolute-version/commit-and-tag-version) fork of `standard-version`.

Is upgrading Renovate users to `commit-and-tag-version` the best thing to do? The maintainer of `standard-version` recommends `release-please`. And the Renovate bot project uses Semantic Release.

It sounds to me that the correct thing to do, as a user of `standard-version` is to move to a tool like `release-please` or Semantic Release?

I'll let the Renovate maintainers decide what to do with my feedback. 😄

Reply:
commit-and-tag-version maintainer here-

Yes; this is a safe thing to do- release please was recommended by the original maintainer of standard-version, but the two tools have different purposes. Release Please is only for users who have GitHub repositories, and additionally it assumes a lot more about how you do your releases.

Semantic release isn’t an appropriate replacement for standard version either - again, the tools do different things. You can’t drop one in for another; you’d have to change your whole release process.

The fork that I maintain was made specifically for those who don’t do their releases in the way that release-please expects, or who are otherwise unable to use github.

Reply:
Thanks for the tag, @ST-DDT !

Reply:
Users that did not move away from `standard-version` to `release-please` and ignoring the deprecation error that always shows up by npm are not moving to `release-please` anyway 🤷 
But they might not have heard of `commit-and-tag-version`